I have not yet updated these tests:

```
FAILED tests/pipelines/test_pipelines_document_question_answering.py::DocumentQuestionAnsweringPipelineTests::test_large_model_pt_chunk - AssertionError: Lists differ: [{'score': 0.9974, 'answer': '1110212019', 'start': 23, [69 chars] 16}] != [{'score': 0.9967, 'answer': '1102/2019', 'start': 22, '[67 chars] 15}]
FAILED tests/pipelines/test_pipelines_document_question_answering.py::DocumentQuestionAnsweringPipelineTests::test_large_model_pt_layoutlm_chunk - AssertionError: Lists differ: [{'sc[39 chars]t': 16, 'end': 16}, {'score': 0.9998, 'answer'[31 chars] 16}] != [{'sc[39 chars]t': 15, 'end': 15}, {'score': 0.9924, 'answer'[3...
```

as I want to inspect the CI failures first. I think both of these tests are caused by tesseract OCR errors (specifically I think the CI is running a diff. version of tesseract than my local machine).

--- a/src/transformers/pipelines/document_question_answering.py
+++ b/src/transformers/pipelines/document_question_answering.py
@@ -235,7 +235,6 @@ class DocumentQuestionAnsweringPipeline(ChunkPipeline):
               `word_boxes`).
             - **answer** (`str`) -- The answer to the question.
             - **words** (`list[int]`) -- The index of each word/box pair that is in the answer
-            - **page** (`int`) -- The page of the answer
         """
         if isinstance(question, str):
             inputs = {"question": question, "image": image}
@@ -315,7 +314,6 @@ class DocumentQuestionAnsweringPipeline(ChunkPipeline):
                 "p_mask": None,
                 "word_ids": None,
                 "words": None,
-                "page": None,
                 "output_attentions": True,
                 "is_last": True,
             }
@@ -339,6 +337,7 @@ class DocumentQuestionAnsweringPipeline(ChunkPipeline):
                 return_overflowing_tokens=True,
                 **tokenizer_kwargs,
             )
+            encoding.pop("overflow_to_sample_mapping")  # We do not use this
 
             num_spans = len(encoding["input_ids"])
 
@@ -395,9 +394,6 @@ class DocumentQuestionAnsweringPipeline(ChunkPipeline):
         words = model_inputs.pop("words", None)
         is_last = model_inputs.pop("is_last", False)
 
-        if "overflow_to_sample_mapping" in model_inputs:
-            model_inputs.pop("overflow_to_sample_mapping")
-
         if self.model_type == ModelType.VisionEncoderDecoder:
             model_outputs = self.model.generate(**model_inputs)
         else:
@@ -421,7 +417,7 @@ class DocumentQuestionAnsweringPipeline(ChunkPipeline):
         return answers
 
     def postprocess_encoder_decoder_single(self, model_outputs, **kwargs):
-        sequence = self.tokenizer.batch_decode(model_outputs.sequences)[0]
+        sequence = self.tokenizer.batch_decode(model_outputs["sequences"])[0]
 
         # TODO: A lot of this logic is specific to Donut and should probably be handled in the tokenizer
         # (see https://github.com/huggingface/transformers/pull/18414/files#r961747408 for more context).

--- a/tests/pipelines/test_pipelines_document_question_answering.py
+++ b/tests/pipelines/test_pipelines_document_question_answering.py
@@ -208,19 +208,13 @@ class DocumentQuestionAnsweringPipelineTests(unittest.TestCase, metaclass=Pipeli
         outputs = dqa_pipeline(image=image, question=question, top_k=2)
         self.assertEqual(
             nested_simplify(outputs, decimals=4),
-            [
-                {"score": 0.9967, "answer": "1102/2019", "start": 22, "end": 22},
-                {"score": 0.996, "answer": "us-001", "start": 15, "end": 15},
-            ],
+           [{'score': 0.9974, 'answer': '1110212019', 'start': 23, 'end': 23}, {'score': 0.9948, 'answer': 'us-001', 'start': 16, 'end': 16}],
         )
 
         outputs = dqa_pipeline({"image": image, "question": question}, top_k=2)
         self.assertEqual(
             nested_simplify(outputs, decimals=4),
-            [
-                {"score": 0.9967, "answer": "1102/2019", "start": 22, "end": 22},
-                {"score": 0.996, "answer": "us-001", "start": 15, "end": 15},
-            ],
+            [{'score': 0.9974, 'answer': '1110212019', 'start': 23, 'end': 23}, {'score': 0.9948, 'answer': 'us-001', 'start': 16, 'end': 16}],
         )
 
         outputs = dqa_pipeline(
@@ -228,13 +222,7 @@ class DocumentQuestionAnsweringPipelineTests(unittest.TestCase, metaclass=Pipeli
         )
         self.assertEqual(
             nested_simplify(outputs, decimals=4),
-            [
-                [
-                    {"score": 0.9967, "answer": "1102/2019", "start": 22, "end": 22},
-                    {"score": 0.996, "answer": "us-001", "start": 15, "end": 15},
-                ]
-            ]
-            * 2,
+            [[{'score': 0.9974, 'answer': '1110212019', 'start': 23, 'end': 23}, {'score': 0.9948, 'answer': 'us-001', 'start': 16, 'end': 16}], [{'score': 0.9974, 'answer': '1110212019', 'start': 23, 'end': 23}, {'score': 0.9948, 'answer': 'us-001', 'start': 16, 'end': 16}]]
         )
 
     @slow
@@ -319,10 +307,7 @@ class DocumentQuestionAnsweringPipelineTests(unittest.TestCase, metaclass=Pipeli
         outputs = dqa_pipeline(image=image, question=question, top_k=2)
         self.assertEqual(
             nested_simplify(outputs, decimals=4),
-            [
-                {"score": 0.9999, "answer": "us-001", "start": 15, "end": 15},
-                {"score": 0.9924, "answer": "us-001", "start": 15, "end": 15},
-            ],
+            [{'score': 0.9999, 'answer': 'us-001', 'start': 16, 'end': 16}, {'score': 0.9998, 'answer': 'us-001', 'start': 16, 'end': 16}],
         )
 
         outputs = dqa_pipeline(
@@ -330,13 +315,7 @@ class DocumentQuestionAnsweringPipelineTests(unittest.TestCase, metaclass=Pipeli
         )
         self.assertEqual(
             nested_simplify(outputs, decimals=4),
-            [
-                [
-                    {"score": 0.9999, "answer": "us-001", "start": 15, "end": 15},
-                    {"score": 0.9924, "answer": "us-001", "start": 15, "end": 15},
-                ]
-            ]
-            * 2,
+            [[{'score': 0.9999, 'answer': 'us-001', 'start': 16, 'end': 16}, {'score': 0.9998, 'answer': 'us-001', 'start': 16, 'end': 16}], [{'score': 0.9999, 'answer': 'us-001', 'start': 16, 'end': 16}, {'score': 0.9998, 'answer': 'us-001', 'start': 16, 'end': 16}]]
         )
 
         word_boxes = list(zip(*apply_tesseract(load_image(image), None, "")))
@@ -345,10 +324,7 @@ class DocumentQuestionAnsweringPipelineTests(unittest.TestCase, metaclass=Pipeli
         outputs = dqa_pipeline({"image": None, "word_boxes": word_boxes, "question": question}, top_k=2)
         self.assertEqual(
             nested_simplify(outputs, decimals=4),
-            [
-                {"score": 0.9999, "answer": "us-001", "start": 15, "end": 15},
-                {"score": 0.9924, "answer": "us-001", "start": 15, "end": 15},
-            ],
+            [{'score': 0.9999, 'answer': 'us-001', 'start': 16, 'end': 16}, {'score': 0.9998, 'answer': 'us-001', 'start': 16, 'end': 16}]
         )
 
     @slow

--- a/tests/pipelines/test_pipelines_document_question_answering.py
+++ b/tests/pipelines/test_pipelines_document_question_answering.py
@@ -208,13 +208,19 @@ class DocumentQuestionAnsweringPipelineTests(unittest.TestCase, metaclass=Pipeli
         outputs = dqa_pipeline(image=image, question=question, top_k=2)
         self.assertEqual(
             nested_simplify(outputs, decimals=4),
-           [{'score': 0.9974, 'answer': '1110212019', 'start': 23, 'end': 23}, {'score': 0.9948, 'answer': 'us-001', 'start': 16, 'end': 16}],
+            [
+                {"score": 0.9974, "answer": "1110212019", "start": 23, "end": 23},
+                {"score": 0.9948, "answer": "us-001", "start": 16, "end": 16},
+            ],
         )
 
         outputs = dqa_pipeline({"image": image, "question": question}, top_k=2)
         self.assertEqual(
             nested_simplify(outputs, decimals=4),
-            [{'score': 0.9974, 'answer': '1110212019', 'start': 23, 'end': 23}, {'score': 0.9948, 'answer': 'us-001', 'start': 16, 'end': 16}],
+            [
+                {"score": 0.9974, "answer": "1110212019", "start": 23, "end": 23},
+                {"score": 0.9948, "answer": "us-001", "start": 16, "end": 16},
+            ],
         )
 
         outputs = dqa_pipeline(
@@ -222,7 +228,11 @@ class DocumentQuestionAnsweringPipelineTests(unittest.TestCase, metaclass=Pipeli
         )
         self.assertEqual(
             nested_simplify(outputs, decimals=4),
-            [[{'score': 0.9974, 'answer': '1110212019', 'start': 23, 'end': 23}, {'score': 0.9948, 'answer': 'us-001', 'start': 16, 'end': 16}], [{'score': 0.9974, 'answer': '1110212019', 'start': 23, 'end': 23}, {'score': 0.9948, 'answer': 'us-001', 'start': 16, 'end': 16}]]
+            [
+                {"score": 0.9974, "answer": "1110212019", "start": 23, "end": 23},
+                {"score": 0.9948, "answer": "us-001", "start": 16, "end": 16},
+            ]
+            * 2,
         )
 
     @slow
@@ -307,7 +317,10 @@ class DocumentQuestionAnsweringPipelineTests(unittest.TestCase, metaclass=Pipeli
         outputs = dqa_pipeline(image=image, question=question, top_k=2)
         self.assertEqual(
             nested_simplify(outputs, decimals=4),
-            [{'score': 0.9999, 'answer': 'us-001', 'start': 16, 'end': 16}, {'score': 0.9998, 'answer': 'us-001', 'start': 16, 'end': 16}],
+            [
+                {"score": 0.9999, "answer": "us-001", "start": 16, "end": 16},
+                {"score": 0.9998, "answer": "us-001", "start": 16, "end": 16},
+            ],
         )
 
         outputs = dqa_pipeline(
@@ -315,7 +328,11 @@ class DocumentQuestionAnsweringPipelineTests(unittest.TestCase, metaclass=Pipeli
         )
         self.assertEqual(
             nested_simplify(outputs, decimals=4),
-            [[{'score': 0.9999, 'answer': 'us-001', 'start': 16, 'end': 16}, {'score': 0.9998, 'answer': 'us-001', 'start': 16, 'end': 16}], [{'score': 0.9999, 'answer': 'us-001', 'start': 16, 'end': 16}, {'score': 0.9998, 'answer': 'us-001', 'start': 16, 'end': 16}]]
+            [
+                {"score": 0.9999, "answer": "us-001", "start": 16, "end": 16},
+                {"score": 0.9998, "answer": "us-001", "start": 16, "end": 16},
+            ]
+            * 2,
         )
 
         word_boxes = list(zip(*apply_tesseract(load_image(image), None, "")))
@@ -324,7 +341,10 @@ class DocumentQuestionAnsweringPipelineTests(unittest.TestCase, metaclass=Pipeli
         outputs = dqa_pipeline({"image": None, "word_boxes": word_boxes, "question": question}, top_k=2)
         self.assertEqual(
             nested_simplify(outputs, decimals=4),
-            [{'score': 0.9999, 'answer': 'us-001', 'start': 16, 'end': 16}, {'score': 0.9998, 'answer': 'us-001', 'start': 16, 'end': 16}]
+            [
+                {"score": 0.9999, "answer": "us-001", "start": 16, "end": 16},
+                {"score": 0.9998, "answer": "us-001", "start": 16, "end": 16},
+            ],
         )
 
     @slow

